msg/sensor_gyro_control.msg

```diff
@@ -0,0 +1,9 @@
+uint32 device_id	# unique device ID for the sensor that does not change between power cycles
+
+uint64 timestamp	# time since system start (microseconds)
+
+uint64 timestamp_sample	# time the raw data was sampled (microseconds)
+
+float32 x		# filtered angular velocity in the NED X board axis in rad/s
```

Comment:
Could we use an array e.g. `float32[3] angular_velocity` in this new message?
That would facilitate processing the data in bulk and we could omit all the three liners for copies or calculations.

Reply:
This was just a quick incremental step to enable flight testing.

Where I'd actually like to get to is having a system wide `vehicle_angular_velocity` message handled by an aggregator outside of the rate controller. https://github.com/dagar/Firmware/commit/edbe24579c56b67258b08b539c2f3f2e586b4fab#diff-b6dea6f43a77ecf4383a55b7ed51818eR2

Reply:
Done.